prime-router/settings/organizations.yml

```diff
@@ -26,6 +26,73 @@
       topic: covid-19
       schemaName: waters/waters-covid-19
       format: CSV
+  receivers:
```

Comment:
This is basically copied from Staging.   Giang was using these two receivers to send to, in his jmeter tests.   I copied them here so that we can do similar tests on localhost.